docs/Deploying-User-Data-Store.md

```diff
@@ -31,7 +31,7 @@ There are two roles, that you may assign to the user in the table `uds_authoriti
 ## Deploying User Data Store
 
 Right now, the only standalone war is supported running from console using the `java -jar` command.
-The deployed application is accessible on `http://localhost:8080/user-data-store/`.
+The deployed application is accessible on `http://localhost:8080/user-data-store-server/`.
```

Comment:
🤔 @zcgandcomp Isn't it a problem for the existing customers? I know, this is „only“ documentation, but now I see the consequences of splitting into modules.

Reply:
Yes, it is. 
We will have to be careful with the migration guidelines and prepare for a guided migration of customers under support.

Reply:
There is a simple and quick fix by setting the property `server.servlet.context-path`.

Reply:
I will fix it in a separate ticket: https://github.com/wultra/user-data-store/issues/170